components/dropdown/dropdown-opener-mixin.js

```diff
@@ -119,7 +119,8 @@ export const DropdownOpenerMixin = superclass => class extends superclass {
 		if (!opener) {
 			return;
 		}
-		opener.setAttribute('aria-expanded', (content && content.opened || false).toString());
+		const attribute = opener.tagName.toUpperCase().startsWith('D2L-BUTTON') ? 'control-expanded' : 'aria-expanded';
```

Comment:
I think we need to consider more than just `d2l-button`. Consumers can use any of the button implementations. For instance, we have the pre-canned `d2l-dropdown-button-subtle`, `d2l-dropdown-context-menu`, `d2l-dropdown-more` openers that use `d2l-button-icon` and `d2l-button-subtle`.